bloom/bloomfilter.go

```diff
@@ -44,22 +47,7 @@ func NewBloomFilterLegacy(m, h uint) (BloomFilter, error) {
 	}
 }
 
-// BloomFilterFromBytesLegacy constructs a bloom filter from bytes
-func BloomFilterFromBytesLegacy(b []byte, m, h uint) (BloomFilter, error) {
```

Comment:
this fun is still needed for legacy support, search the name in iotex-core codebase you will see